server/app/lib/api_analytics_tracer.rb

```diff
@@ -8,7 +8,8 @@ def analyze_query(query: )
     params = {
       user_ip: query.context[:request_ip],
       query: query.query_string.squish,
-      query_variables: query.provided_variables
+      query_variables: query.provided_variables,
+      client_type: request.headers['dgidb-client-name']
```

Comment:
I haven't tested it myself, but I don't believe `request` is available in here by default. You may have to grab this value in the controller and pass it in via the query context (like the ip address).

Reply:
this is what I get for forcing a PR out before lunch